addons/mail/static/src/components/chatter/chatter.scss

```diff
@@ -9,6 +9,10 @@
     overflow: auto;
 }
 
+.o_form_sheet .o_Chatter {
+  overflow: visible;
+}
```

Comment:
Why we need `overflow: auto;` on `.o_Chatter` again? @seb-odoo

Reply:
I've been asked the same question from AAB 2 months ago in the context of another problem.

If we can't' see anything wrong without it we should remover it.

Reply:
So to answer the question: I don't know.

Reply:
Ok, so I would remove `overflow: auto;` from `.o_Chatter` and check nothing's broken.
Should check community/enterprise and some special cases of chatter usage, like in Documents app.